app/src/main/java/com/distillery/interview/util/Constants.kt

```diff
@@ -0,0 +1,8 @@
+package com.distillery.interview.util
+
+object Constants {
```

Comment:
Instead of having a generic `Constants` object you can create an `companion object` within each class to have the information in the specific places.  For example:
```
WeatherCollectionAdapter {

   companion object {
      private const val VIEWPAGER_CANT_TABS = 3
   }
}
```

Reply:
I could create companion objects but in this case I need to use these const variables in multiples clases, not only in Adapter!

Reply:
Having a `Constants` file is old-school 😛. You can create a companion object in the adapter and declare private and public constants.

Reply:
That sounds better! 😉